src/components/Text/text/text.ts

```diff
@@ -28,6 +28,15 @@ export const TEXT_VARIANTS = [
     'code-inline-1',
 ] as const;
 
+const TEXT_FONT_WEIGHT_VARIANTS = [
```

Comment:
I guess design system support only limites set of font  weight

Reply:
And typically value is a number (100-900)

Reply:
adding numeric value is not the problem. I'll do it soon

We need to understand what is the legitimate way to add the `font-weight` property to this component, not taking in mind what we can pass css class with right font-weight in `className` prop - it is provides very ugly dev exp.

Alternative way is have different `variants` for every variant: for example `body-1-bold` but it seems that this is the way to madness.